lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -20,6 +20,8 @@ defmodule Mix.Tasks.App.Tree do
     * `--pretty` - use Unicode codepoints for formatting the tree.
       Defaults to true except on Windows.
 
+    * `--dot` - produces a DOT graph description of the application tree.
```

Comment:
That's a very good question. I like separate options because they are more descriptive and easier to document. Although we can have incompatible options (like passing both pretty and dot).